CommandCenter.m

```diff
@@ -184,21 +181,24 @@ function CommandCenter_OpeningFcn(hObject, eventdata, handles, varargin)
     set(handles.file_logger,'checked',handles.logger.visible)
     
     % Convert panels to scrollPanels
+    set(textH,'String','Making Panels'); drawnow;
     loaded_vars = load(fullfile(path,'static','reload_icon.mat'));
     handles.reload_CData = loaded_vars.im;
     handles.panelStage = Base.UIscrollPanel(handles.panelStage);
     handles.panelImage = Base.UIscrollPanel(handles.panelImage);
     handles.panelSource = Base.UIscrollPanel(handles.panelSource);
     handles.panelExperiment = Base.UIscrollPanel(handles.panelExperiment);
     handles.panelSave = Base.UIscrollPanel(handles.panelSave);
-    controls = [handles.panelStage,handles.panelImage,handles.panelSource,...
-        handles.panelExperiment,handles.panelSave];
+%     handles.panelGit = Base.UIscrollPanel(handles.panelGit, false);
```

Comment:
why is this commented?

Reply:
As a reminder that panelGit isn't a UIscrollPanel or Resizable, but I removed it.